docs/src/pages/landing-page/Index.vue

```diff
@@ -101,7 +109,7 @@
             Get all you need to setup your project basic
             <span class="block">features and save time</span>
           </div>
-          <q class="lp-heading--quote">He who controls Scaffolding, controls the universe</q>
+          <p class="lp-heading--quote">He who controls Scaffolding, controls the universe</p>
```

Comment:
Oh, so you didn't mean to use `q` tag in the first place? My previous comment about this wasn't ironic, I didn't know there was a `q` tag, but it seems the semantically correct one to use here

Reply:
Yes, I'm also just discovering there's a `q` tag after reading this feedback. Coincidentally, it really seems to be the best to use in this case.